server/routers/index.js

```diff
@@ -1,10 +1,13 @@
 import { Router } from "express";
 import userRouter from "./user.router.js";
 import roleRouter from "./role.router.js";
+import authRouter from "./auth.router.js";
+import jwtMiddleware from "../middleware/jwt.middleware.js";
 
 const router = new Router();
 
-router.use("/user", userRouter);
+router.use("/user", jwtMiddleware, userRouter);
```

Comment:
It's better to add middlewares to specific endpoints not to groups. 
But it's OK to wrap groups with error handlers, parsers and middlewares that don't abort request processing